fc/mef.py

```diff
@@ -607,14 +353,11 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
     ----------
     data_beads : FCSData object
```

Comment:
Not for now. The domain is used in a couple of places. It would not be hard to change it, but it will require testing, so this would probably need its own issue.